### DISCUSSION TITLE
mem_limit限制无效

### DISCUSSION THREAD

**Discussion by stgztsw** [apache/doris#19657] [A - General / Q&A]
doris版本1.0.0
be.conf配置了mem_limit=55%的限制。但是还是因为内存使用过多，被linux内核杀了。感觉mem_limit没有起到效果

**Comment by zy-kkk** [comment]
这个版本很久了，可以考虑升级最新的1.2.4.1
因为Doris不能跨2版本升级，所以需要先升级1.1.5，然后升级1.2.4.1
